graphql_server/aiohttp/graphqlview.py

```diff
@@ -162,7 +159,21 @@ async def __call__(self, request):
             )
 
             if is_graphiql:
-                return await self.render_graphiql(params=all_params[0], result=result)
+                graphiql_data = GraphiQLData(  # type: ignore
```

Comment:
Why the `type: ignore`?

Reply:
Because mypy will complain about `GraphiQLData` missing fields, which are being passed by the `**` operator.